### ISSUE TITLE
Pretty-print the JSON in the project builder

### ISSUE THREAD

**Issue by pdehaan** [loads/loads-web#47]
Currently, we're displaying the JSON blob, but it'd be nice if it was a bit prettier (but i'm sure the JSON is pretty on the _inside_).

### Current output:
```json
{"project_title":"project title","plans":[{"plan_title":"plan 1 title","plan_description":"plan 1 desc","steps":[{"step_name":"","instance_count":"4","run_max_time":"4000","run_delay":"267","step_url":"","environment_data":"","dns_mapping":"","port_mapping":"","volume_mapping":"","docker_series":""}]}]}
```

### Better output:
```json
{
  "project_title": "project title",
  "plans": [
    {
      "plan_title": "plan 1 title",
      "plan_description": "plan 1 desc",
      "steps": [
        {
          "step_name": "",
          "instance_count": "4",
          "run_max_time": "4000",
          "run_delay": "267",
          "step_url": "",
          "environment_data": "",
          "dns_mapping": "",
          "port_mapping": "",
          "volume_mapping": "",
          "docker_series": ""
        }
      ]
    }
  ]
}
```

#### Protip: `JSON.stringify(data, null, 2)`

There may be an Angular module for prettier JSON, but the above `stringify()` should be a quick fix.

**Comment by pdehaan** [comment]
Possible Bower modules if we don't want to roll our own: http://bower.io/search/?q=ng-json

**Comment by pdehaan** [comment]
Left a comment in https://github.com/loads/loads-web/pull/46/files#r24284482 as well since that code hasn't technically landed yet.

**Comment by pdehaan** [comment]
Closing, fixed by #56
